openwisp_controller/tests/utils.py

```diff
@@ -1,6 +1,6 @@
 from django.contrib.auth import get_user_model
 
-from openwisp_utils.tests.utils import TestMultitenantAdminMixin
+from openwisp_users.tests.utils import TestMultitenantAdminMixin
```

Comment:
I'm not sure why this is needed, since this change has been already done on master: https://github.com/openwisp/openwisp-controller/commit/74600dc85f3ff0d4383c96c54bc00db122292a22

Have you rebased this branch on the current master?

Reply:
You're right, thats not needed